### PR TITLE
RMP-4: feature: Action and delete modals, menu's and form

### PR DESCRIPTION
**Pull request by verkhovsky**
Screenshot 2021-03-25 at 11 02 53
Screenshot 2021-03-25 at 11 02 57
Screenshot 2021-03-25 at 11 03 08
Screenshot 2021-03-25 at 11 02 29

### CODE REVIEW

File: src/components/FormFields/SelectField/SelectField.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  id,
+  name,
+  placeholder,
+  label,
+  className,
+  ...props
+}) => {
+  return (
+    <div className={classnames('select-field--wrapper', className)}>
+      {!!label && <Label label={label} />}
```
Comment (Andrew-Bogolubof) [inline src/components/FormFields/SelectField/SelectField.jsx:19]: `!!label` -> `label` No need to use `!!`

File: src/pages/Home/components/Card/Card.jsx [hunk 2]
```diff
@@ -1,29 +1,92 @@
...
+    }
+  }, []);
+
+  useEffect(() => {
+    if (isMenuOpen) {
+      document.addEventListener('mousedown', handleOutsideClick);
+    } else {
+      document.removeEventListener('mousedown', handleOutsideClick);
+    }
+  }, [isMenuOpen, handleOutsideClick]);
```
Comment (Andrew-Bogolubof) [inline src/pages/Home/components/Card/Card.jsx:52]: What if we move to another page? We will have a memory leak here. Use effect should remove listener additionally in the function returned from the `useEffect`.

File: src/components/FormFields/SelectField/SelectField.jsx (hunk 1 above)
Comment (verkhovsky) [reply src/components/FormFields/SelectField/SelectField.jsx:19]: fixed in next PR

File: src/pages/Home/components/Card/Card.jsx (hunk 2 above)
Comment (verkhovsky) [reply src/pages/Home/components/Card/Card.jsx:52]: also fixed in next PR

**Pull request merged**